Description: Use walinux-agent rather than builtin fabric support
 Upstream now uses the built-in support for instance initialization on Azure.
 On a stable release, we want to continue to use the walinux-agent integration.
 Upstream made this change under bug 1538522.
Forwarded: not-needed
Author: Scott Moser <smoser@ubuntu.com>
--- a/cloudinit/sources/DataSourceAzure.py
+++ b/cloudinit/sources/DataSourceAzure.py
@@ -196,7 +196,7 @@ if util.is_FreeBSD():
         LOG.debug("resource disk is None")
 
 BUILTIN_DS_CONFIG = {
-    'agent_command': AGENT_START_BUILTIN,
+    'agent_command': AGENT_START,
     'data_dir': AGENT_SEED_DIR,
     'set_hostname': True,
     'hostname_bounce': {
